examples/eng_wasm_contracts/millionaires_problem_demo/src/lib.rs

```diff
@@ -0,0 +1,88 @@
+#![no_std]
+#![feature(proc_macro_gen)]
+#![feature(proc_macro_non_items)]
+
+
+extern crate eng_wasm;
+extern crate eng_wasm_derive;
+extern crate rustc_hex as hex;
+#[macro_use]
+extern crate serde_derive;
+extern crate serde;
+
+use eng_wasm::*;
+use eng_wasm_derive::pub_interface;
+use eng_wasm::String;
+use eng_wasm::from_utf8;
+use hex::ToHex;
```

Comment:
It seems like FromHex and ToHex are not used, therefore lines 17 and 18 may be deleted.